ambari-server/src/main/java/org/apache/ambari/server/security/authentication/AmbariAuthenticationEventHandlerImpl.java

```diff
@@ -96,15 +97,18 @@ public void onUnsuccessfulAuthentication(AmbariAuthenticationFilter filter, Http
     String username;
     String message;
     String logMessage;
+    String proxyUser;
     Integer consecutiveFailures = null;
     boolean incrementFailureCount;
 
     if (cause == null) {
       username = null;
       message = "Unknown cause";
+      proxyUser = null;
       incrementFailureCount = false;
     } else {
       username = cause.getUsername();
+      proxyUser = RequestUtils.getQueryStringParameterValue(servletRequest, "doAs");
```

Comment:
Maybe add proxyuser info to `AmbariAuthenticationException` rather than trying to get it from the request.

Reply:
addressed in next commit